parsers/to-css-textStyle/to-css-textStyle.spec.ts

```diff
@@ -0,0 +1,121 @@
+import seeds from '../../seeds';
+import { default as toCssTextStyle, InputDataType } from './to-css-textStyle.parser';
+import libs, { LibsType } from '../global-libs';
+import { TextStyleValue } from '../../types';
+
+describe('to-css-text-style', () => {
+  it('Get tokens - execute parser', async done => {
+    const result = await toCssTextStyle(
+      (seeds().tokens.filter(({ type }) => type === 'textStyle') as unknown) as InputDataType,
+      undefined,
+      libs,
+    );
+    if (result instanceof Error) return done.fail(result);
+    expect(typeof result).toEqual('string');
+    expect(
+      result.includes(
+        '.ts-with-allan-again {\n' +
```

Comment:
Is this normal ".ts-with-allan-again"?

Reply:
What's the problem ?

Reply:
I don't see a problem per se. I just wanted to check if having this string `.ts-with-allan-again` was correct. Because when I see "Allan" I don't know if you're talking of @AllanMichay or the font family named "Alan".

If you're referencing the font family then I'd like you to rename it to "Alan" so we don't misunderstand.
If you're referencing @AllanMichay I'd like to know why.

Reply:
It's just the name of the text style.